corehq/apps/accounting/static/accounting/js/pricing_table.js

```diff
@@ -19,39 +31,57 @@ hqDefine('accounting/js/pricing_table', function () {
         self.selectCurrentPlan = function () {
             self.selected_edition(self.currentEdition);
         };
+        self.capitalizeString = function (s) {
+            return s.charAt(0).toUpperCase() + s.slice(1);
+        };
+        self.isDowngrade = function (oldPlan, newPlan) {
+            if (oldPlan === 'Enterprise') {
```

Comment:
This file could benefit from named constants, it'd protect against typos in the plan names.